service/src/main/kotlin/io/provenance/explorer/domain/entities/Blocks.kt

```diff
@@ -286,6 +286,21 @@ class BlockCacheHourlyTxCountsRecord(id: EntityID<DateTime>) : Entity<DateTime>(
             }
         }
 
+        fun getTxHeatmap() = transaction {
+            val blockTimestamp = BlockCacheHourlyTxCountsTable.blockTimestamp
+            val txCount = BlockCacheHourlyTxCountsTable.txCount
+            BlockCacheHourlyTxCountsTable
+                .slice(blockTimestamp, txCount)
```

Comment:
This query needs to be summed up to the hour level.

Reply:
It's using `block_cache_hourly_tx_counts` table that is used by `Tx History`. The table is already summed up to the hour level.

The only difference between `heatmap` and `tx history` is that we don't set limits on `heatmap`. Should there be more slicing to the `heatmap` results?

Reply:
Brain fart. You are correct. :nothing-to-see-here:

Reply:
So you have it returning a list of hourly records, but they need to be summed across day of the week. Brain fart on the hourly trunc already, but thats only half of it.

Reply:
Added `DAY` granularity